src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -189,6 +189,9 @@ public function setHeaders(array $headers): static
         return $this;
     }
 
+    /**
+     * @return $this
+     */
     public function setRows(array $rows)
```

Comment:
we could also add a static return type

Reply:
As the class is not final, we cannot do it outside a major version